### PR TITLE
Add option to force usage of FQN

### LINKED ISSUE #1007: Disable "Simplify full namespaces"
Hi,

How to disable "Simplify full namespaces for already included resources" and keep FQCN like in version 2.7?

### PR DESCRIPTION
**Pull request by edvordo**
Closes #1007

### CODE REVIEW

File: src/Console/ModelsCommand.php [hunk 1]
```diff
@@ -1067,8 +1067,9 @@ protected function getClassNameInDestinationFile(object $model, string $classNam
         $className = trim($className, '\\');
         $writingToExternalFile = !$this->write;
         $classIsNotInExternalFile = $reflection->getName() !== $className;
+        $forceFQCN = $this->laravel['config']->get('ide-helper.force_fqn', false);
```
Comment (mfn) [inline src/Console/ModelsCommand.php:1070]: I suggest to stay consistent regarding the abbreviation:
```suggestion
        $forceFQN = $this->laravel['config']->get('ide-helper.force_fqn', false);
```

**Review by mfn — APPROVED** [review]
Besides the var name, LGTM!

**Comment by UksusoFF** [comment]
@mfn seems getUsedClassNames method also need check for force_fqn?

**Comment by mfn** [comment]
@UksusoFF 🤷‍♀️ I don't use that option. Can you create a PR with tests?

**Comment by UksusoFF** [comment]
Seems it's some unexpected behavior with my code. Will check it.

**Comment by mfn** [comment]
Okay, thanks for checking 👍 Please keep us updated!

**Pull request merged**